docs/basics/acct-team-mgmt/concurrency-limits.md

```diff
@@ -6,17 +6,45 @@ sidebar_label: Concurrency Limits and Team Accounts
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-If your organization has multiple teams sharing a Sauce Labs account, it's important to monitor your concurrency limits--the number of virtual machines that you can run tests on simultaneously. You'll need to ensure that your organization's concurrency is distributed among your accounts, rather than allocating all of it to a single user.
+
+If your organization has multiple teams sharing a Sauce Labs account, it's important to monitor your concurrency limits--the number of virtual machines that you can run tests on simultaneously. You'll need to ensure that your organization's concurrency is distributed among your accounts.
+
+If you are an org admin, you can view your organization's concurrency limits:
+1. In Sauce Labs, click **ACCOUNT** and then click **Team Management**.
+
+  <img src={useBaseUrl('img/team-mgmt/team-mgmt-nav.png')} alt="Team management navigation" width="300"/>
+
+2. On the **Organization Management** page, under the ORGANIZATION NAME box, concurrency limits will be displayed for the selected data center. You can also view the **PEAK VM CONCURRENCY** and **VM TESTS BY DAY** for the organization.
+
+  <img src={useBaseUrl('img/team-mgmt/ccy-limits-org-mgmt.png')} alt="Org management concurrency limits" width="600"/>
+
+3. To view the limits for another data center, in the upper-right corner of the window, click the **DATA CENTER** dropdown and select the relevant data center.
+
+  <img src={useBaseUrl('img/team-mgmt/ccy-data-center-dropdown.png')} alt="Data Center dropdown" width="250"/>
+
+4. On the **Organization Management** page, click the **TEAMS** tab. **Team VM Concurrency** and **Peak VM Concurrency** will be displayed for each team in the organization.
+
+  <img src={useBaseUrl('img/team-mgmt/ccy-limits-teams.png')} alt="Team concurrency limits" width="600"/>
+
+For more information about usage, see [Viewing and Exporting Usage Data](https://docs.saucelabs.com/basics/acct-team-mgmt/viewing-exporting-usage-data/).
 
 ## How Concurrency Limits Work
-Only organization admins can set the total concurrency of their organization's Sauce Labs account. When setting the concurrency limit for a team or user, be mindful that by default, each team member will inherit the concurrency limits of that entire team. If there are enough members on that team, their combined concurrency allocation could easily consume the concurrency allocation for the entire organization.
+Through Sauce Labs, organization admins set the total concurrency of their organization's account. When setting the concurrency limit for a team, be mindful that team members share the concurrency limit of the team. If a team exceeds its concurrency limit, any additional tests will be queued (see [Queueing Tests](#queueing-tests) for more information).
```

Comment:
This is a little unclear to me. The first sentence says the org admin set the "total ccy for the org", which to me, sort of implies that the org admin does "not" control the individual team settings...but then the next sentence talks about setting the limit for a team. So I find myself wondering 1) does the org admin also control the individual team ccy allowances, and 2) can a team admin also set ccy for their own team, or no because that would potentially take ccy _away_ from other teams? Maybe the first sentence could say something like "Only the org admin can allocate portions of the organization's total ccy to each team within the organization."